compose-files/docker-compose.yml

```
@@ -34,7 +34,7 @@ services:
     command: --bind_ip_all --replSet rs0
 
   mongodb-secondary:
-    image: mongo:5.0.26 ## change tag in version.env
+    image: mongo:6.0.15
```

Last version after docker hub is 6.0.16 
```suggestion
    image: mongo:6.0.16
```